### PR TITLE
Update layers.md

### PR DESCRIPTION
**Pull request by SivanSrimanPC**
L87 to L90

1)the function declaration was missing the self argument, so added it.

2)i changed the color changing method call to just changing the hue parameter of the rgb object... this works perfectly with the breathing animation

### CODE REVIEW

**Comment by xs5871** [comment]
> 2)i changed the color changing method call to just changing the hue parameter of the rgb object... this works perfectly with the breathing animation

But it doesn't work with the static animation, because that one doesn't periodically update the LEDs.

**Comment by SivanSrimanPC** [comment]
> > 2)i changed the color changing method call to just changing the hue parameter of the rgb object... this works perfectly with the breathing animation
> 
> But it doesn't work with the static animation, because that one doesn't periodically update the LEDs.

oh then i guess we can add this as a separate example for animations then
but for both cases the self argument needs to be passed

**Comment by xs5871** [comment]
Easy fix: leave the `rgb.set_hsv_fill(...)` line in. I agree with the rest of the PR.

File: docs/en/layers.md [hunk 1]
```diff
@@ -84,10 +84,10 @@ class Layers(_Layers):
 	last_top_layer = 0
 	hues = (4, 20, 69)
 	
-	def after_hid_send(keyboard):
+	def after_hid_send(self, keyboard):
 		if keyboard.active_layers[0] != self.last_top_layer:
 			self.last_top_layer = keyboard.active_layers[0]
-			rgb.set_hsv_fill(self.hues[self.last_top_layer], 255, 255)
+			rgb.hue = self.last_top_layer
```
Comment (xs5871) [inline docs/en/layers.md:90]:
```suggestion
			rgb.set_hsv_fill(self.hues[self.last_top_layer], 255, 255)
			rgb.hue = self.hues[self.last_top_layer]
```

**Review by xs5871 — CHANGES_REQUESTED** [review]

**Comment by xs5871** [comment]
There's a pending change request still open. Any updates?

**Review by xs5871 — APPROVED** [review]

**Pull request merged**